### ISSUE TITLE
Cannot find module 'object-assign-shim'

### ISSUE THREAD

**Issue by 4393HE** [olmokramer/atom-minimap-codeglance#29]
Error: Cannot find module 'object-assign-shim'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/张光明/.atom/packages/atom-minimap-codeglance/lib/minimap-codeglance.js:156:42)
    at Module._compile (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .js] (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (C:\Users\张光明\.atom\packages\atom-minimap-codeglance\lib\main.js:10:28)
    at Module._compile (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .js] (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package.js:760:27)
    at C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package.js:121:28
    at Package.module.exports.Package.measure (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package.js:94:15)
    at Package.module.exports.Package.load (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package.js:108:12)
    at PackageManager.module.exports.PackageManager.loadPackage (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package-manager.js:466:14)
    at C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package-manager.js:410:19
    at Config.module.exports.Config.transact (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\config.js:312:16)
    at PackageManager.module.exports.PackageManager.loadPackages (C:\Users\张光明\AppData\Local\atom\app-1.13.0-beta6\resources\app.asar\src\package-manager.js:405:19)

**Comment by davisonio** [comment]
How did you install the package? Using GUI or apm? object-assign-shim should have been installed as a dependancy.

**Comment by olmokramer** [comment]
Hi everyone, I'm sorry for the extremely late reply. As you have probably noticed I've stopped maintaining my Atom plugins a few years ago. I got very frustrated with how Atom was maintained and started ignoring all of my plugins. I'm sorry for that. I won't install Atom to test and mergeer PRs, but I'd be happy to transfer my repositories to anyone that wants to maintain any of my plugins. Again, I'm sorry.
